tests/python/frontend/keras/test_forward.py

```diff
@@ -227,6 +227,7 @@ def test_forward_activations_except(self, keras_mod):
             act_funcs = [
                 keras_mod.layers.LeakyReLU(alpha=None),
                 keras_mod.layers.LEU(2, 3, 4),
+                keras_mod.layers.ReLU(threshold=None),
```

Comment:
Is this behavior actual only for Keras <= 2.3.1?

Reply:
Yes, When Keras > 2.3.1, an assertion have been added to the Keras to crash this usage. Thus, this test is only for Keras <= 2.3.1